templates/zerver/register.html

```diff
@@ -26,6 +26,20 @@ <h1>You're almost there.</h1>
         <form method="post" class="form-horizontal white-box" id="registration" action="{{ url('zerver.views.registration.accounts_register') }}">
             {{ csrf_input }}
 
+            {% if realm_name %}
+            <div class="split-view" >
+                <div class="left-side">
+                    <div class="org-header reg-org-header">
```

Comment:
Do we need three nested divs here, or can we combine the first two?

Reply:
Actually, it allows me to use the previous css written for some other pages without writing new one. Since, the css is written is such a way that nested divs are required so I've to use it.

@showell let me know your views about this and tell me about the changes so that I can finish the work related to this PR and get it merged.

Reply:
@VishalCR7 Yeah, that's a perfectly good reason to keep the code as you wrote it.  Thanks for the explanation.